core/src/main/scala/org/http4s/Static.scala

```diff
@@ -0,0 +1,212 @@
+package org.http4s
+
+
+import java.io._
+import java.net.URL
+
+
+import cats.implicits.{catsSyntaxEither => _, _}
+import cats.data.OptionT
+import cats.effect.{ContextShift, Sync}
+import fs2.Stream.empty
+import fs2.io._
+import fs2.io.file.readRange
+import org.http4s.headers._
+import org.http4s.Status.NotModified
+import org.log4s.getLogger
+
+import scala.concurrent.ExecutionContext
+
+
+trait Static[F[_]] {
+
+  def calcETag(f: File): F[String]
+
+  def fromString(
+    url: String,
+    blockingExecutionContext: ExecutionContext,
+    req: Option[Request[F]] = None): OptionT[F, Response[F]]
+
+  def fromResource(
+    name: String,
+    blockingExecutionContext: ExecutionContext,
```

Comment:
Why isn't the buffer size parameterized here, like for `fromFile`?

Reply:
Nvm, I see where it's used.